src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -228,165 +223,158 @@ All reconstruction specific parameters that are not specified, but one wants to
+| PlasmaFreeFractionMethod            | RECOMMENDED if `PlasmaAvail` is `true`  | [string][]    | Method used to estimate free fraction.                                                                                                                                                                               |
+| MetaboliteMethod                    | REQUIRED if `MetaboliteAvail` is `true` | [string][]    | Method used to measure metabolites.                                                                                                                                                                                  |
+| MetaboliteRecoveryCorrectionApplied | REQUIRED if `MetaboliteAvail` is `true` | [boolean][]   | Metabolite recovery correction from the HPLC, for tracers where it changes with time postinjection. If `true`, the `metabolite_recovery_fractions` column SHOULD be present in the corresponding `*_blood.tsv` file. |
+
+The following columns are defined for `_blood.tsv` files:
+
+| **Column name**              | **Requirement level**                                          | **Description**                                                                   | **Units**                                                                         |
+| ---------------------------- | -------------------------------------------------------------- | --------------------------------------------------------------------------------- | --------------------------------------------------------------------------------- |
+| `time`                       | REQUIRED                                                       | Time in relation to `TimeZero` defined by the `*_pet.json`. (for example, 5)      | Seconds                                                                           |
+| `plasma_radioactivity`       | REQUIRED if `PlasmaAvail` is `true`                            | Radioactivity in plasma                                                           | Unit of plasma radioactivity (for example, "kBq/mL")                              |
+| `metabolite_parent_fraction` | REQUIRED if `MetaboliteAvail` is `true`                        | Parent fraction of the radiotracer (0-1)                                          | Unit of parent fraction (for example, "unitless")                                 |
+| `metabolite_polar_fraction`  | RECOMMENDED if `MetaboliteAvail` is `true`                     | Polar metabolite fraction of the radiotracer (0-1)                                | Unit of polar metabolite fraction (for example, "unitless")                       |
+| `hplc_recovery_fractions`    | RECOMMENDED if `MetaboliteRecoveryCorrectionApplied` is `true` | HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC) | Unit of recovery fractions (for example, "unitless")                              |
```

Comment:
@mnoergaard Should this one be changed too?

```suggestion
| `hplc_recovery_fractions`    | REQUIRED if `MetaboliteRecoveryCorrectionApplied` is `true` | HPLC recovery fractions (the fraction of activity that gets loaded onto the HPLC) | Unit of recovery fractions (for example, "unitless")                              |
```

Reply:
Yes! Thanks, @effigies.